src/apps/bridge/aanderaaController.cpp

```diff
@@ -69,6 +69,16 @@ static constexpr uint32_t SAMPLE_TIMER_MS = 30 * 1000;
 static constexpr uint32_t TOPO_TIMEOUT_MS = 10 * 1000;
 static constexpr uint32_t NODE_INFO_TIMEOUT_MS = 1000;
 
+#define PI  3.14159265358979323846
+
+static constexpr double DIRECTION_SAMPLE_MIN = 0.0;
+static constexpr double DIRECTION_SAMPLE_MAX = 2*PI;
+static constexpr double ABS_SPEED_SAMPLE_MIN = 0.0;
+static constexpr double ABS_SPEED_SAMPLE_MAX = 300.0;
+static constexpr double TEMP_SAMPLE_MIN = -5.0;
+static constexpr double TEMP_SAMPLE_MAX = 40.0;
```

Comment:
Change these from SAMPLE to READING.

Reply:
OOH yeah, these shouldn't be SAMPLE. SAMPLE_MEMBER may be more appropriate since these are being used to check the values that are going into the sample, not each individual reading?

Reply:
It's debatable since ideally these ought to be the ranges for the readings as well as the sample members... but since we saw that the aanderaa was giving negative direction readings and can translate that into the sample, I take it back — let's just leave these as-is.